### PR TITLE
Support session timeout

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: it/helpers/ComponentSpecBase.scala [hunk 1]
```diff
@@ -271,4 +271,10 @@ trait ComponentSpecBase extends PlaySpec with CustomMatchers with GuiceOneServer
   def removeHtmlMarkup(stringWithMarkup: String): String =
     stringWithMarkup.replaceAll("<.+?>", " ").replaceAll("[\\s]{2,}", " ").trim
 
+  def sessionTimeout(): WSResponse = get("/session-timeout")
+
+  def agentSessionTimeout(): WSResponse = get("/client/session-timeout")
+
+
+
```
Comment (AlexRimmerHMRC) [inline it/helpers/ComponentSpecBase.scala:279]: No longer needed

File: test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/services/agent/MockAgentAuthService.scala [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/*
```
Comment (AlexRimmerHMRC) [inline test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/services/agent/MockAgentAuthService.scala:1]: Not used, please remove

File: it/helpers/ComponentSpecBase.scala (hunk 1 above)
Comment (llmikeyj) [reply it/helpers/ComponentSpecBase.scala:279]: Done

File: test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/services/agent/MockAgentAuthService.scala (hunk 2 above)
Comment (llmikeyj) [reply test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/services/agent/MockAgentAuthService.scala:1]: Done

File: app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/config/UrlHelpers.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.config
+
+trait UrlHelpers {
+
+  def maybeClient(path: String): String = if (path.contains("/client")) "client/" else ""
```
[the remark this answers is not in the document]
Comment (llmikeyj) [reply app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/config/UrlHelpers.scala:21]: Done

**Pull request merged**